phlib/theme.c

```diff
@@ -799,6 +816,7 @@ BOOLEAN CALLBACK PhpThemeWindowEnumChildWindows(
             //case 1: // Old colors
             PhWindowThemeSetDarkMode(tooltipWindow, TRUE);
             PhWindowThemeSetDarkMode(WindowHandle, TRUE);
+            AllowDarkModeForWindow_I(WindowHandle, TRUE);
```

Comment:
This path is crashing for me on 1809. `AllowDarkModeForWindow_I` is `NULL`.

Reply:
Moved to phlib, see `PhAllowDarkModeForWindow`, `PhIsDarkModeAllowedForWindow`.

Reply:
Go ahead and commit this change then I think we can merge 👍

Reply:
Ok, I added some finishing touches to Object Manager and cleanup code [9c589e6](https://github.com/winsiderss/systeminformer/pull/2197/commits/9c589e6b7718eb11820e94f9a3f54996b5545304)

Add support for KernelObject and KernelOnlyAccess attributes. I think this is quite usefull. NtQueryObject only returns OBJ_EXCLUSIVE and OBJ_PERMANENT.
Please check the driver code, hope this implementation is correct :wink:

https://github.com/user-attachments/assets/1397c45d-7818-4ba2-bf40-4fce2fa418ab